patterns/cta-pricing.php

```diff
@@ -26,6 +26,10 @@
 	</div>
 	<!-- /wp:group -->
 
+	<!-- wp:spacer {"height":"var:preset|spacing|50"} -->
+	<div style="height:var(--wp--preset--spacing--50)" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
```

Comment:
Because the spacer block has the default top margin of a nested block, spacing preset 50 on the block is larger than the previous bottom padding.

Left is the current, right is the proposed change in the PR:

![image](https://github.com/user-attachments/assets/47750727-bcb2-4dd9-b27f-99959563a512)

Reply:
Thanks, Carolina. I added the changes accounting for that margin.